### PR TITLE
Server side render rich embeds

### PR DESCRIPTION
**Pull request by Charrondev**
Allows server rendering of embed blots. Test fixtures have been added for all current types.

This includes:

- LinkEmbed
- ImageEmbed
- VideoEmbed

Loading and error blots have been excluded as they don't show up on the server.

I've disabled HtmlLawed and opened this issue https://github.com/vanilla/vanilla/issues/6652 because it was stripping too much.

### CODE REVIEW

**Comment by slafleche** [comment]
Code looks good. Let's make a test page in the rich-editor as a spot check. Although you can create one yourself, it's good to have a version controlled test.

**Comment by Charrondev** [comment]
The test page has been accessed. It can be found at `/richeditorstyles/renderer`.

File: plugins/rich-editor/views/richeditorstyles/rendered.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...linkImage":"https://cdn.mdn.mozilla.net/static/img/opengraph-logo.72382e605ce3.png","excerpt":"The Battle of New York, locally known as \"The Incident\", was a major battle between the Avengers and Loki with his borrowed Chitauri army in Manhattan, New York City. It was, according to Loki\'s plan, the first battle in Loki\'s war to subjugate Earth, but the actions of the Avengers neutralized the threat of the Chitauri before they could continue the invasion."}}},{"insert":"External Link Embed "},{"attributes":{"header":2},"insert":"\n"},{"insert":"\n"},{"insert":{"link-embed":{"url":"https://www.google.ca/","name":"Hulk attacks New York, kills 17, injures 23 in deadliest attack in 5 years   Hulk attacks New York, kills 17, injures 23 in deadliest attack in 5 years","source":"nytimes.com","excerpt":"The Battle of New York, locally known as \"The Incident\", was a major battle between the Avengers and Loki with his borrowed Chitauri army in Manhattan, New York City. It was, according to Loki\'s plan, the first battle in Loki\'s war to subjugate Earth, but the actions of the Avengers neutralized the threat of the Chitauri before they could continue the invasion."}}},{"insert":"\n\n"}]
```
Comment (slafleche) [inline plugins/rich-editor/views/richeditorstyles/rendered.php]: Get rid of `{"insert":"\n"},`, it looks bad to have more space after the title than before.

**Comment by slafleche** [comment]
There's also a function to make the video playable on click, let's hook that up

**Pull request merged**